pkg/bincapz/bincapz.go

```diff
@@ -53,3 +54,13 @@ type Report struct {
 	Diff   DiffReport            `json:",omitempty" yaml:",omitempty"`
 	Filter string                `json:",omitempty" yaml:",omitempty"`
 }
+
+type Kv struct {
```

Comment:
Can you pick a name that makes its use more obvious?

I was thinking something like "MetricInt" or "StatInt" but I'm sure there are better ideas here.

Reply:
Addressed in [`5ad17ab` (#121)](https://github.com/chainguard-dev/bincapz/pull/121/commits/5ad17abd03cd920455ed146562cb0bf3c9ac992f).